bridge-monitor/README.md

```diff
@@ -1,12 +1,18 @@
-# Bridge Monitor
-Deployed version:
-https://bridge-monitoring.poa.net/
+# POA Token Bridge / Monitor
+Tool for checking balances and unprocessed events in bridged networks.
+
+## Overview
+Please refer to the [POA Token Bridge](../README.md) overview first of all.
+
+- Deployed version: https://bridge-monitoring.poa.net/
 
 This tools allows you to spin up node.js server to monitor for 2 contracts on
```

Comment:
Please change text on line 9 to this:
This tool initiates a node.js server which can monitor balance differences between the Home and Foreign Eth networks.
On the Home network it checks the `HOME_BRIDGE_ADDRESS` balance.	
On the Foreign network it checks the `ERC20 token` total supply for the `FOREIGN_BRIDGE_ADDRESS`.

Reply:
@andogro since the token bridge is working in several modes the monitor could check not only the account balance on the Home side but the token balance as well (for `erc20-to-erc20`) mode. Moreover the monitor also checks consistency in the validators list on both sides and other things.
That's why I would recommend to generalize description as follows: `This tool allows you to spin up a NODE.JS server to monitor for health of the Token Bridge contracts: check for the balance difference, discover incosistency in the validators list, catch unhandled transactions.`. The same I made on [the monitor repo](https://github.com/poanetwork/bridge-monitor/)

Reply:
I changed the description to the latter. @akolotov Please be aware that there is a typo in `incosistency`. I have changed it to `inconsistency` here but it remains wrong on the monitor repo.